core/beacon/gen_blockparams.go

```diff
@@ -15,14 +15,23 @@ var _ = (*payloadAttributesMarshaling)(nil)
 // MarshalJSON marshals as JSON.
 func (p PayloadAttributesV1) MarshalJSON() ([]byte, error) {
 	type PayloadAttributesV1 struct {
-		Timestamp             hexutil.Uint64 `json:"timestamp"     gencodec:"required"`
-		Random                common.Hash    `json:"random"        gencodec:"required"`
-		SuggestedFeeRecipient common.Address `json:"suggestedFeeRecipient"  gencodec:"required"`
+		Timestamp             hexutil.Uint64  `json:"timestamp"     gencodec:"required"`
+		Random                common.Hash     `json:"random"        gencodec:"required"`
```

Comment:
As of kiln v2, the `Random` field is now `prevRandao` in the JSON
https://github.com/ethereum/go-ethereum/commit/4f4622bc8b104fc87c5ed64ed86895fe118e1ea7

Reply:
That change is not yet included in this version of geth we are building on.

Reply:
We should probably just rebase the commit on the latest geth to get all bug-fixes, and then make these minor changes in an update to the rollup-node.

Reply:
Fixed in #8, please continue rollup review comments there.